tox.ini

```diff
@@ -5,6 +5,7 @@ envlist =
     py{36,37,38,39,py3}-django22-alchemy-mongoengine
     py{36,37,38,39,py3}-django30-alchemy-mongoengine
     py{36,37,38,39,py3}-django31-alchemy-mongoengine
+    py{36,37,38,39,py3}-django32-alchemy-mongoengine
```

Comment:
@francoisfreitag Have you checked the support policy for Django 3.2 w.r.t Python versions?

Reply:
Yup: https://docs.djangoproject.com/en/dev/faq/install/#what-python-version-can-i-use-with-django.